<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>72</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>1/2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="percentLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="markLabel">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>330</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Your Result</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="viewAnswersButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>View answers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainMenuButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
